pkg/proxy/volume.go

```diff
@@ -247,3 +253,24 @@ func (p *Proxy) spdkVolumeUnmapMarkSnapChainRemovedSet(ctx context.Context, req
 	/* Not implemented */
 	return &empty.Empty{}, nil
 }
+
+func (p *Proxy) VolumeMountStatusCheck(ctx context.Context, req *rpc.VolumeMountStatusRequest) (resp *rpc.VolumeMountStatusResponse, err error) {
+	if req.VolumeName == "" {
+		return nil, grpcstatus.Error(grpccodes.InvalidArgument, "volume name is required")
+	}
+
+	errMsg := ""
+
+	globalMountPath := fmt.Sprintf(types.VolumeGlobalMountPath, sha256.Sum256([]byte(req.VolumeName)))
```

Comment:
What does it append with a checksum?

Reply:
The global mount path format is as follow
```
/var/lib/kubelet/plugins/kubernetes.io/csi/<csi driver name>/<sha256sum of pv.spec.csi.volumeHandle>/globalmount
```